source/CIColor.h

```diff
@@ -29,9 +29,9 @@ class CIColor {
         purple,
         red,
         yellow,
+        turquoise,
```

Comment:
do we want blue or turquoise to be the 4th color in the game. because if we go with turquoise we should give it a lock in powerup